### PR TITLE
[RND-344] Documentation updates in preparation for 0.2.0.

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

File: docs/TECHNICAL.md [hunk 1]
```diff
@@ -1,73 +1,103 @@
...
+pair representing the natural key components as defined by the Ed-Fi Data
+Standard. Example: `{"studentUniqueId":"abc"}`. These are then combined as:
+
+```none
+hash(Ed-Fi#Student) + hash({"studentUniqueId":"abc"})
+```
+
+The hash values are further base 64 encoded for string storage, leading to a
+38-character string. For example: `WhT14ozRv-M80122NydSim1PK4FiQIxmPRXMTA`
+corresponds to the student document described above.
```
Comment (bradbanister) [inline docs/TECHNICAL.md:28]: Good catch. Forgot this was in here.

File: docs/README.md [hunk 2]
```diff
@@ -53,9 +59,13 @@ Meadowlark is packaged with the full set of Ed-Fi descriptors which, while not
...
-* In a second shell, cd into /Meadowlark/eng and the run Invoke-Bulkload.ps1 script
-  * Note: This requires building the [Bulk Load Client Utility](https://techdocs.ed-fi.org/display/ODSAPIS3V53/Bulk+Load+Client+Utility)
-* If using AWS-lambda as your service layer, you can also run `yarn load:descriptors:local` from the /Meadowlark-js/services/meadowlark-aws-lambda folder to load descriptors
+* In a second shell, cd into /Meadowlark/eng and the run Invoke-Bulkload.ps1
+  script
+  * Note: This requires building the [Bulk Load Client
+    Utility](https://techdocs.ed-fi.org/display/ODSAPIS3V53/Bulk+Load+Client+Utility)
+* If using AWS-lambda as your service layer, you can also run `yarn
+  load:descriptors:local` from the /Meadowlark-js/services/meadowlark-aws-lambda
+  folder to load descriptors
```
Comment (bradbanister) [inline docs/README.md:68]: Needs removal

File: docs/LOCALHOST.md [hunk 3]
```diff
@@ -6,15 +6,25 @@ Instructions for running a local "developer" environment on localhost:
...
-   * **PostgreSQL**: Start PostgreSQL in Docker ([instructions](../Meadowlark-js/backends/meadowlark-postgresql-backend/docker/readme.md)).
-6. Initialize OpenSearch by starting it up in Docker
-   ([instructions](../Meadowlark-js/backends/meadowlark-opensearch-backend/docker/readme.md)).
-7. Setup environment variables for running [meadowlark-fastify](../Meadowlark-js/services/meadowlark-fastify/)
-   service. The folder has an example.env file with all settings needed to run the service, the easiest way to set your environment variables is to duplicate this file in the folder and rename the file to .env
-   1. Review the settings in the .env file to see what values, if any, to change. In particular:
-      * DOCUMENT_STORE_PLUGIN - Make sure your chosen backend is uncommented and comment out the rest
-      * QUERY_HANDLER_PLUGIN - Having this value uncommented allows for querying from Opensearch
-      * LISTENER1_PLUGIN - Having this value uncommented will allow Meadowlark documents to flow to OpenSearch
+5. The Meadowlark runtime currently requires running both either PostgreSQL or
```
Comment (andonyns) [inline docs/LOCALHOST.md:9]: @stephenfuqua Is this **both either** a typo on the instructions?

Comment (stephenfuqua) [reply docs/LOCALHOST.md:9]:
```suggestion
5. The Meadowlark runtime currently requires running either PostgreSQL or
```

**Review by bradbanister — APPROVED** [review]
PR looks good

**Pull request merged**